backend/src/packages/chaiNNer_standard/image_utility/compositing/blend_images.py

```diff
@@ -23,18 +77,116 @@
         ImageInput("Base Layer", channels=[1, 3, 4], allow_colors=True),
         ImageInput("Overlay Layer", channels=[1, 3, 4], allow_colors=True),
         BlendModeDropdown(),
+        if_group(Condition.type(0, "Image") & Condition.type(1, "Image"))(
+            EnumInput(
+                BlendOverlayPosition,
+                label="Overlay position",
+                option_labels=BLEND_OVERLAY_POSITION_LABELS,
+                default_value=BlendOverlayPosition.CENTERED,
```

Comment:
CI is failing because I changed the name of this parameter to `default` today.

```suggestion
                default=BlendOverlayPosition.CENTERED,
```